project/en-US/rts/integrating_with_other_transaction_models.xml

```diff
@@ -55,6 +55,12 @@ public class JAXRSResource {
     }
 }
             </programlisting>
+
+            <para>
+                REST-AT transaction context does not provide timeout.
+                When REST-AT transaction is bridged to JTA then the imported JTA transaction
```

Comment:
The JTA transaction is not imported, rather, a new one is created and connected to the incoming REST-AT transaction. I would just replace the word imported with bridged.

Reply:
ok, I changed the wording for bridged